### DISCUSSION TITLE
15 kelime güvenlik için uygunmu

### DISCUSSION THREAD

**Discussion by 8nokta2** [IBAX-io/go-ibax#1102] [Soru Cevap - Türkçe] [answered]
bu 15 kelimeyi kaybettiğimiz takdirde giremiyormuyuz kesinlikle cüzdanımıza

**Comment by Yusufknk** [comment, marked as answer]
Hayır girebilirsiniz. 15 Kelime ve şifreyi yeni cüzdan oluştur kısmından girerek ulaşabilirsiniz.
